main.py

```diff
@@ -8,6 +8,14 @@
 import datetime
 import traceback
 
+# Script for exporting database
+# As before using MONGODB_URI in env to act as original database
+# Add MONGODB_URI_ATLAS in env to act as target database
+import export_database
+export_database.export_db();
+# export_database.insert_from_file("defaulters.bson");
+
+
```

Comment:
I think as we discussed, let's run this locally with the two DB URI - old one and new one. What do you say?

Reply:
Ohh I started to build to to push to heroku but if we're running it locally then we'll just kill the script after the export is completed. I should have mentioned it in the steps. I'll add a better control to the script!